app/service_utils.py

```diff
@@ -30,6 +31,24 @@ def update_and_validate_service(service, service_payload):
 
 
 def validate_service(service):
+
+    # TODO: Get framework slug from Framework table once it exists.
```

Comment:
There's a few places this can be done now the Frameworks are in.  I don't think it needs to be in this PR, which I am sick of the sight of.